ihatemoney/models.py

```diff
@@ -262,6 +262,14 @@ def get_member_bills(self, member_id):
             .order_by(Bill.id.desc())
         )
 
+    def get_newest_bill(self):
+        """Returns the most recent bill (according to bill date) or None if there are no bills"""
+        return self.get_bills_unordered().order_by(Bill.date.desc()).first()
```

Comment:
I am not familiar enough with SQLAlchemy. Could you check that it doesn't fetch a full batch of records from the database? Otherwise, we could manually add a `LIMIT 1` to the query for obvious performance reasons.

Reply:
Good idea, I checked and sqlalchemy correctly adds a `LIMIT` itself. I added a comment to make that clear.